server/business-process-query/get-data.js

```diff
@@ -110,10 +138,76 @@ module.exports = exports = memoize(function (driver) {
 			});
 		});
 
+		storage.on('update', function (event) {
```

Comment:
This handling should be in `get-status-history-date-map.js ` and should update data on currentDate on _dates map_